src/markdown-pages/add-ons/kotsadm.md

```diff
@@ -22,10 +24,26 @@ spec:
     uiBindPort: 8800
     hostname: "hostname"
     applicationNamespace: "kots"
+    disableS3: true
 ```
 
 flags-table
 
+## Side Effects of the `disableS3` flag
```

Comment:
I wonder if removing the word "Side" here would sound better. Side effects connote something unintentional, so I wonder if "Effects of the `disableS3` flag" would work better.

Reply:
I like this suggestion!